### PR TITLE
Newtest

### PR DESCRIPTION
**Pull request by aanchalcoder18**
homepage redesign

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -1,92 +1,110 @@
...
-      <button onclick="showPanel(2,'#4caf50')">Monthly View</button>
+  <!-- end of top section-->
+
+  <div class="col-sm">
+    <h1 class="new">
+      New to ThinQ.tv?
+    </h1>
+    <h3 class="Subtitle">
+      Most of the day we have a video chat
+      <%= link_to "Activism Hall", studyhall_path, style: "color: #ffc627" %>
```
Comment (Hieu-Minh-Phung) [inline app/views/static_pages/home.html.erb:31]: I think this should not be "Activism Hall" and Lisa advised that the name of the buttons/links should not be hard-coded.

**Comment by kmyee97** [comment]
**Y means pass**
1. Sign Up works localhost:3000/signup Y
2. Sign In works localhost:3000/signin Y
3. Check if new event works (3 events, past/present/future) Y
4. Profile info page works localhost:3000/profileinfo Y
5. Profile section of controlpanel works Y
            a. Make sure changes are saved Y
            b. Can connect to Stripe Y
            c. Can upload images and save changes Y
6. Create rewards page works Y
            a. Create Buy Y
            b. Create Donate Y
            c. Upload image and provide other needed descriptions Y
7. Check shows part of controlpanel page Y
8. Check account part of controlpanel page Y
            a. Change email works Y
            b. Change password works Y
            c. Change URL and make sure all changes save Y
9. Check Sales part of controlpanel page Y
10. Check Viewprofile page Y
11. Check that links in footer works Y

_The redesign looks good. However, there are some problems on the redesign._

**First of all, please make sure to have the present text.**
Screen Shot 2020-08-10 at 12 41 14 PM

**Secondly, some items are not aligned in regards to the countdown clock/text to picture.**
Screen Shot 2020-08-10 at 1 08 02 PM

**Thirdly, the countdown clock, text, and pictures overlap when the browser window shrinks in terms of width. (I was running this on my MacBook Pro 13 in. I feel this problem only occurs for laptop users and not Desktop users.)**

Screen Shot 2020-08-10 at 12 36 31 PM

**Finally, the monthly calendar view shows a list view of the events. The monthly calendar is gone.**

Screen Shot 2020-08-10 at 12 21 38 PM

**In conclusion, I will not recommend merging this to the master branch.**

**Comment by ece106** [comment]
Big ugly mess. We can't have this on the home page.

**Comment by vinhnguyen200891337** [comment]
The layout in homepage looks weird. You need some kind of text or button in the Discover Activism that makes you banner. The New to ThinQ.tv? need to move to the right a little bit. The countdown clock's text looks kinda bland. Also there is too much space between the drop in button in the dropin hours pic. Also the caption underneath the pics need to be realigned to the right or something so that it looks better. The Monthly View of the calendar does not work. It just shows the conversation. The footer text are not aligned properly. The footer in the Control Panel (specifically the Rewards. Sales, Blocked Users) has too much bottom padding.

File: app/views/static_pages/home.html.erb (hunk 1 above)
Comment (ece106) [reply app/views/static_pages/home.html.erb:31]: We certainly can't have hardcoded links. However, I looked at home.html.erb and did not see hardcoded links

**Comment by vinhnguyen200891337** [comment]
Screenshot from 2020-08-10 15-09-13
Screenshot from 2020-08-10 15-09-21
Screenshot from 2020-08-10 15-09-23
Screenshot from 2020-08-10 15-09-38

**Comment by vinhnguyen200891337** [comment]
Sorry, if I sound like nit-picking

**Comment by ece106** [comment]
What do you mean the month view doesnt work

**Comment by ece106** [comment]
Be more specific

**Comment by ece106** [comment]
@kmyee97 Please debug the monthly calendar

**Comment by kmyee97** [comment]
@vinhnguyen200891337  the calendar for the homepage should only show the conversation events. It shouldn't show any drop-in or group problem-solving events.

**Here is a zoomout screenshot of the homepage calendar with non recurring and recurring conversation events**
Screen Shot 2020-08-10 at 3 56 45 PM
